libr/anal/fcn.c

```diff
@@ -368,6 +368,13 @@ static const char *retpoline_reg(RAnal *anal, ut64 addr) {
 		const char *thunk = strstr (flag->name, token);
 		if (thunk) {
 			return thunk + strlen (token);
+		} else {
+			token = "llvm_retpoline_r";
+			thunk = strstr (flag->name, token);
+			if (thunk) {
+				// r + release number digits
+				return thunk + strlen (token) + 2;
```

Comment:
If you have thunk = "llvm_retpoline_r" without additional chars, this would return a pointer beyond the end of the string, wouldn't it?

Reply:
There is 2 digits after as `llvm_retpoline_r11`